Makefile

```diff
@@ -7,6 +7,16 @@ RDMD_FLAGS=--force --build-only --compiler=$(D_COMPILER) $(D_FLAGS)
 
 PLATFORM := $(shell uname -s)
 
+LDMD=ldmd2
+
+# force this to run on every make. make eats newlines.
+LDC_VERSION_STRING := $(shell $(LDMD) --version | head -1)
+LDC_DMD_LLVM_VERSION_STRING := $(shell $(LDMD) --version | head -2 | tail -1)
+LDC_LDC_VERSION_STRING := $(shell $(LDMD) --version | head -3 | tail -1)
```

Comment:
Version of LDC/DMD the compiler has been built with appeared in LDC 1.0.0 and I doubt it could be of any interest to the users. I'd propose to skip it.